collections.md

```diff
@@ -7,10 +7,10 @@
 - [Multiple Collection Methods in a Row](#multiple-collection-methods-in-a-row)
 - [Calculate Sum with Pagination](#calculate-sum-with-pagination)
 - [Serial no in foreach loop with pagination](#serial-no-in-foreach-loop-with-pagination)
```

Comment:
This deletion is not mentioned in your commit or PR. It is similar but different from the tip that follows it and should probably be kept. 

```suggestion
- [Serial no in foreach loop with pagination](#serial-no-in-foreach-loop-with-pagination)
- [Higher order collection methods](#higher-order-collection-methods)
```